app/styles/app.scss

```diff
@@ -1385,45 +1384,94 @@ section {
       //margin-top: 40px;
 
       .external-video {
-        height: calc(100vh)!important;
+        height: calc(100vh - 140px) !important;
       }
     }
 
-    .section-heading {
-      position: absolute;
-      padding: 10px;
-      height: 65px;
-      color: white;
-      width: 100%;
-      font-size: 38px;
-      background: #363636;
-      display: flex;
-      padding-left: 10px;
+    &:hover {
+      .section-heading {
+        display: flex;
+      }
+    }
+  }
 
-      p {
-        min-width: 95%;
-        font-size: 20px;
-        margin: 14px 0;
+  .section-heading {
+    position: fixed;
+    top: 94px;
+    padding: 10px 20px;
+    color: white;
+    background: #363636;
+    padding-left: 10px;
+    z-index: 9999;
+    font-size: 18px;
+    transition: .5s;
+
+    &.full-width-header {
+      width: 100% !important;
+      max-width: 100vw;
+      overflow: hidden;
+      transition: .5s;
+    }
+
+    .title-text {
+      padding: 15px;
+      display: inline-block;
+      @media (max-width: 767px) {
```

Comment:
This is a wrong media query @abhishek97 
No class is defined on which this should be applied.

Also it is written inside a class which is wrong.